pkg/benchmarkapi/benchmarkapi_suite_test.go

```diff
@@ -147,28 +137,37 @@ func Flood(counter *int64, queriers int) {
 	atomic.StoreInt64(&stop, 1)
 }
 
-var _ = FDescribe("How many calls can we get through in 1 seconds", func() {
+var _ = Describe("How many calls can we get through in 1 seconds", func() {
 	// Want to get some idea of what throughput looks like
+	flag.StringVar(&outputFile, "outFile", fmt.Sprintf("./output-%v.csv", time.Now().Unix()),
```

Comment:
To make it a bit less hacky, you can declare another anonymous variable immediately before, and assign a `func() {}` that does the flag parsing, and add a comment why this is needed (to avoid any heart attacks of future readers).